pointcloud/src/main/scala/geotrellis/pointcloud/raster/rasterize/triangles/PDALTrianglesRasterizer.scala

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.pointcloud.raster.rasterize.triangles
+
+import geotrellis.raster.{DoubleArrayTile, GridBounds, GridExtent, Tile}
+import geotrellis.vector.Extent
+import io.pdal.{DimType, PointView, Triangle}
+import io.pdal.pipeline._
+
+object PDALTrianglesRasterizer {
+  def apply(gridExtent: GridExtent[Long], bounds: GridBounds[Long], sourceBounds: GridExtent[Long], pv: PointView): Tile = {
```

Comment:
I didn't think about the interface too much, it was a strightforward copy paste with no regressions noticed.